js/src/components/SEMrushRelatedKeyphrasesModal.js

```diff
@@ -55,38 +56,80 @@ class SEMrushRelatedKeyphrasesModal extends Component {
 		this.props.onClose();
 	}
 
+	/**
+	 * Opens the popup window.
+	 *
+	 * @param {event} e The click event.
+	 *
+	 * @returns {void}
+	 */
+	onLinkClick( e ) {
+		const url = "https://oauth.semrush.com/oauth2/authorize?" +
+			"ref=1513012826&client_id=yoast&redirect_uri=https%3A%2F%2Foauth.semrush.com%2Foauth2%2Fyoast%2Fsuccess&response_type=code&scope=user.id";
+		const height = "570";
+		const width  = "340";
+		const top = window.top.outerHeight / 2 + window.top.screenY - ( height / 2 );
+		const left = window.top.outerWidth / 2 + window.top.screenX - ( width / 2 );
+
+		const features = [
+			"top=" + top,
+			"left=" + left,
+			"width=" + width,
+			"height=" + height,
+			"resizable=1",
+			"scrollbars=1",
+			"status=0",
+		];
+
+		window.open( url, "SEMrush_login", features.join( "," ) );
+		e.preventDefault();
+	}
+
 	/**
 	 * Renders the RelatedKeyPhrasesModal modal component.
 	 *
 	 * @returns {React.Element} The RelatedKeyPhrasesModal modal component.
 	 */
 	render() {
-		const { keyphrase, location, whichModalOpen } = this.props;
+		const { keyphrase, location, whichModalOpen, isLoggedIn } = this.props;
 
 		return (
 			<Fragment>
-				<BaseButton
-					id="yoast-get-related-keyphrases"
-					className="yoast-related-keyphrases-modal__button"
-					onClick={ this.onModalOpen }
-				>
-					{ __( "Get related keyphrases", "wordpress-seo" ) }
-				</BaseButton>
+				{ isLoggedIn &&
+				<div className={ "yoast" }>
```

Comment:
I'd place this on the line above for consistency with the other div at line 122.